components/feature/p2p/src/main/assets/extensions/p2p/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "manifest_version": 2,
+  "name": "Mozilla Android Components - P2P",
+  "version": "1.0",
+  "content_scripts": [
+    {
+      "matches": ["*://*/*"],
+      "js": [ "p2p.js" ],
+      "run_at": "document_idle"
+    }
+  ],
+  "permissions": [
+    "nativeMessaging",
+    "contextMenus",
```

Comment:
Where do we use all of these permissions like `contextMenus`, `downloads`, `history`, etc?

Reply:
I should have cleaned that up after copying from another extension.